### PR TITLE
fix(hlapi): fix safe-deserialization feature gating

### PR DESCRIPTION
**Pull request by mayeul-zama**
Fixes https://github.com/zama-ai/tfhe-rs-internal/issues/373#event-11372892071

### CODE REVIEW

**Review by IceTDrinker — COMMENTED** [review]
Should we add some Makefile target for when the safe deserialize is not active ? or enable the safe serialization at all time ?

**Comment by mayeul-zama** [comment]
> Should we add some Makefile target for when the safe deserialize is not active ? or enable the safe serialization at all time ?

Removing safe serialization feature (enabling it all the time) would be simpler.
But `bincode` can't be an optional dependency anymore then. It's not really a problem IMO.

**Comment by IceTDrinker** [comment]
> > Should we add some Makefile target for when the safe deserialize is not active ? or enable the safe serialization at all time ?
> 
> Removing safe serialization feature (enabling it all the time) would be simpler. But `bincode` can't be an optional dependency anymore then. It's not really a problem IMO.

Yep was thinking about that and bincode

File: tfhe/Cargo.toml [hunk 1]
```diff
@@ -56,12 +54,12 @@ concrete-csprng = { version = "0.4.0", path = "../concrete-csprng", features = [
 lazy_static = { version = "1.4.0", optional = true }
 serde = { version = "1.0", features = ["derive"] }
 rayon = { version = "1.5.0" }
-bincode = { version = "1.3.3", optional = true }
+bincode = "1.3.3"
```
Comment (IceTDrinker) [inline tfhe/Cargo.toml:57]: looks like there is a space after the version specifier ?

On VS Code I like the even better toml extension for formatting the toml automatically via alt + shift + F

Comment (mayeul-zama) [reply tfhe/Cargo.toml]: Done

**Review by IceTDrinker — APPROVED** [review]
Thanks !

**Pull request merged**